packages/utils/src/lib/__snapshots__/report-to-md.spec.ts.snap

```diff
@@ -1,533 +1,440 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
-exports[`report-to-md > should contain all sections when using the fixture report > # Code Pushup Report
+exports[`report-to-md > should contain all sections when using the fixture report 1`] = `
+"# Code PushUp Report
 
----
-_Package Name: @code-pushup/core_
-_Version: 0.0.1_
-_Commit: feat(cli): add logic for markdown report - 7eba125ad5643c2f90cb21389fc3442d786f43f9_
-_Date: 2023-10-18T07:49:45.506Z_
-_Duration: 1652 ms_
-_Plugins: 2_
-_Audits: 52_
----
+|🏷 Category|⭐ Score|🛡 Audits|
+|:--|:--:|:--:|
+|[Performance](#performance)|🟢 **92**|6|
+|[Bug prevention](#bug-prevention)|🟡 **68**|16|
+|[Code style](#code-style)|🟡 **54**|13|
 
+## 🏷 Categories
 
-|Category|Score|Audits|
-|:--:|:--:|:--:|
-|Performance|0.9215000000000001|2/1|
-|Bug prevention|0.6842105263157895|16/19|
-|Code style|0.5384615384615384|13/13|
+### Performance
 
-**Performance 0.9215000000000001**
+Performance metrics [📖 Docs](https://developers.google.com/web/fundamentals/performance)
 
-- <details>
-<summary>Disallow missing \`key\` props in iterators/collection literals (0) ESLint</summary>
-1 warning
-ESLint rule **jsx-key**, from _react_ plugin.
-[Documentation](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-key.md)
+🟢 Score:  **92**
 
-</details>
+- 🟢 Performance (_Lighthouse_)
+  - 🟨 [First Contentful Paint](#first-contentful-paint-lighthouse) - <b>1.2 s</b>
```

Comment:
Nitpick, but in this section we can use plain Markdown.

```suggestion
  - 🟨 [First Contentful Paint](#first-contentful-paint-lighthouse) - **1.2 s**
```

Reply:
Fixed, expanded `getAuditResult()` function